docs/fastdds/security/access_control_plugin/access_control_plugin.rst

```diff
@@ -314,13 +315,17 @@ The following table summarizes the elements and sections that each domain rule m
 |                                       |                                          +-----------------------------------+
 |                                       |                                          | ``true``                          |
 +---------------------------------------+------------------------------------------+-----------------------------------+
-| `Metadata protection Kind`_           | ``<metadata_protection_kind>``           | ``true``                          |
+| `Metadata protection Kind`_           | ``<metadata_protection_kind>``           | ``SIGN``                          |
```

Comment:
Strictly speaking, the  `<metadata_protection_kind>` can take the following values:
`NONE`, `SIGN`, `ENCRYPT`, `SIGN_WITH_ORIGIN_AUTHENTICATION`, `ENCRYPT_WITH_ORIGIN_AUTHENTICATION`